frontend/javascripts/viewer/view/action-bar/dataset_position_view.tsx

```diff
@@ -49,12 +48,6 @@ class DatasetPositionView extends PureComponent<Props> {
     Toast.success("Position copied to clipboard");
   };
 
-  copyRotationToClipboard = async () => {
-    const rotation = V3.round(getRotation(this.props.flycam)).join(", ");
-    await navigator.clipboard.writeText(rotation);
-    Toast.success("Rotation copied to clipboard");
-  };
-
```

Comment:
I removed this feature as it likely was pretty much unused and reusing this button to make a rotation reset seemed far more useful to me